### PR TITLE
Enhancement/1380 remove analytics requirement

### PR DESCRIPTION
**Pull request by ShahAaron**
## Summary

Addresses issue #1380 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by ShahAaron** [comment]
@googlebot I consent.

**Comment by ShahAaron** [comment]
@googlebot I consent.

File: assets/js/components/setup-module.js [hunk 1]
```diff
@@ -106,6 +107,7 @@ class SetupModule extends Component {
 			requiredModules.forEach( ( requiredModule ) => {
 				if ( ! modules[ requiredModule ].setupComplete ) {
 					blockedByParentModule = true;
+					parentModule = requiredModule.replace( /(^\w{1})|(\s{1}\w{1})/g, ( match ) => match.toUpperCase() );
```
Comment (aaemnnosttv) [inline assets/js/components/setup-module.js:110]: We can't simply transform a module slug into a name for display like this as the module name is a translatable string. We need to use `modules[ requiredModule ].name` if it exists.

Comment (ShahAaron) [reply assets/js/components/setup-module.js:110]: @aaemnnosttv if the `modules[ requiredModule ].name` does not exist, is there a fallback ?

Comment (aaemnnosttv) [reply assets/js/components/setup-module.js]: The required module should always exist. It already assumes this above where it checks if `setupComplete` so we can assume it exists here as well.

Comment (ShahAaron) [reply assets/js/components/setup-module.js]: Fixed as requested

**Pull request merged**